<!--xml-->
 <launch>
 <node ns="camera1" name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video1" />
    <param name="image_width" value="1280" />
    <param name="image_height" value="720" />
    <param name="pixel_format" value="mjpeg" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
  </node>
  <node ns="camera3" name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video2" />
    <param name="image_width" value="1280" />
    <param name="image_height" value="720" />
    <param name="pixel_format" value="mjpeg" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
  </node>
  <node pkg="lane_detect" type="leftcam.py" name="leftcam" output="screen"></node>
  <node pkg="lane_detect" type="rightcam.py" name="rightcam" output="screen"></node>
  <node pkg="lane_detect" type="IPM" name="IPM" output="screen"></node>
  <node pkg="lane_detect" type="dist_interpolater" name="dist_interpolator" output="screen"></node>
</launch>
